src/emulator/dataconnect/pgliteServer.ts

```diff
@@ -51,13 +53,22 @@ export class PostgresServer {
       });
     });
     const listeningPromise = new Promise<void>((resolve) => {
+      server.on("error", (e: any) => {
```

Comment:
Do we want to allow users to run both psql on their machine and PGLite?

Maybe the logs should suggest users to change their port instead

Reply:
Good call, forgot i made this configurable - reworded slightly to suggest changing this.